src/utils.js

```diff
@@ -0,0 +1,9 @@
+const getUser = function() {
+  return window.gapi.auth2.getAuthInstance().currentUser.get()
+}
+
+const getAuthToken = function() {
+  return getUser().getAuthResponse().access_token
+}
+
+export { getUser, getAuthToken }
```

Comment:
utils.js seems a bit too generic considering what this contains. I suggest auth.js.